src/main/java/emissary/util/shell/Executrix.java

```diff
@@ -796,50 +797,56 @@ public int execute(final String[] cmd, final byte[] data, final StringBuilder ou
      * @return process exit status
      */
     public int execute(final String[] cmd, @Nullable final byte[] data, @Nullable final StringBuilder out, @Nullable final StringBuilder err,
-            @Nullable final String charset,
-            @Nullable final Map<String, String> env) {
+            @Nullable final String charset, @Nullable final Map<String, String> env) {
+        return execute(new ExecuteConfig.SbExecuteConfig(cmd, data, out, err, charset, env));
+    }
+
+    /**
+     * Executes a command in a new process through Runtime Exec
```

Comment:
It would be helpful if the Javadoc descriptions for the 3 new `execute` overloads gave some insight on when it would be more appropriate to use these instead of the many existing overloads - e.g., to capture binary output.  Binary "mode" isn't an explicit output type like STD or FILE.

Reply:
updated javadocs